system/kube-monitoring/charts/prometheus-frontend/openstack-swift.alerts

```diff
@@ -0,0 +1,184 @@
+ALERT SwiftAccountFirstByteTiming
```

Comment:
Please generalize FirstByteTiming. They are the same for account, container and object.